DUI3-DX/Connectors/Autocad/Speckle.Connectors.AutocadShared/Bindings/AutocadSelectionBinding.cs

```diff
@@ -66,13 +67,19 @@ public SelectionInfo GetSelection()
           }
 
           var handleString = dbObject.Handle.Value.ToString();
+          objectTypes.Add(dbObject.GetType().Name);
           objs.Add(handleString);
         }
 
         tr.Commit();
         tr.Dispose();
```

Comment:
Call to `Dispose` here is unnecessary no? the transaction is within a `using` statement

Reply:
you are right! removing it